lib/letter_opener_web/letter_interface.rb

```diff
@@ -0,0 +1,92 @@
+# frozen_string_literal: true
+
```

Comment:
To be absolutely sure `ActiveSupport::Concern` is loaded - because this is a Gem, and we're currently getting it transiently via action mailer, maybe we should explicitly require it?

```suggestion
require "active_support"
require "active_support/concern"
```